stable/chat/v1/messages.proto

```diff
@@ -537,8 +513,11 @@ message TriggerActionRequest {
   uint64 channel_id = 2;
   // Message ID of the message you want to trigger an action in.
   uint64 message_id = 3;
+  // Data that alllows the bot to discern which action the user is trying to trigger.
+  // A server MUST verify that this info refers to a valid action for the target message.
+  bytes info = 4;
   // Payload of action data.
-  ActionPayload payload = 4;
+  optional bytes payload = 5;
```

Comment:
what's the difference between payload or info?

Reply:
payload: the data generated by the client
info: the data generated by the bot to identify the specific action

Reply:
i was thinking of adding a server-generated action_id to each action but I don't think that would work well, since the bot would still need to figure out which action is which